src/subscription/subscription.service.ts

```diff
@@ -30,8 +33,26 @@ export class SubscriptionService {
     private jwtService: JwtService,
     private httpService: HttpService,
     private readonly analyticsService: AnalyticsService,
+    private readonly configService : ConfigService
   ) {}
 
+  /**
+   * sends an axios post request to the notification service to notify user of new subscriptions
+   * @param apiId : string
+   * @param profileId : string 
+   * @param subscriberId : string
+   * @returns : Axios response object
+   */
+  async makeSubscriptionCall(apiId:string, profileId: string, subscriberId: string): Promise<AxiosResponse<any>>{
```

Comment:
please rename to something better @alahirajeffrey

Reply:
@PhilemonBrain Function has been renamed